docs/modules/ROOT/examples/io/quarkiverse/langchain4j/samples/ChatMemoryProviderBean.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkiverse.langchain4j.samples;
+
+import dev.langchain4j.memory.ChatMemory;
+import dev.langchain4j.memory.chat.ChatMemoryProvider;
+import dev.langchain4j.memory.chat.MessageWindowChatMemory;
+import dev.langchain4j.store.memory.chat.ChatMemoryStore;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+@ApplicationScoped
+public class ChatMemoryProviderBean implements ChatMemoryProvider {
+
+    @Inject
+    ChatMemoryStore store;
+
+    @Override
+    public ChatMemory get(Object memoryId) {
+        return MessageWindowChatMemory.builder()
+                .id(memoryId)
```

Comment:
without eviction or4 cleanup, this will lead to OOM.

Reply:
I need to see exactly how this is meant to work.

Reply:
Yes, there is currently no eviction mechanism. However, for a simple example, that would be hard to do. The current idea of always evicting when the request ends makes a chat with API calls impossible.

Reply:
The documentation should mention or have a more complex example with eviction. 
I'm still unsure about the request scope part. I think, we would need to introduce a "conversation" concept soon.